manifests/default.pp

```diff
@@ -47,10 +47,12 @@
 
 include apache::mod::prefork
 include apache::mod::php
+include apache::mod::rewrite
 
 apache::vhost { 'joechem.dev':
   servername => 'joechem.dev',
-  ip => '0.0.0.0',
-  port    => '80',
-  docroot => '/var/www/public'
+  ip         => '0.0.0.0',
+  port       => '80',
+  docroot    => '/var/www/public',
+  override   =>  ['All'],
```

Comment:
See https://github.com/puppetlabs/puppetlabs-apache#override and https://httpd.apache.org/docs/current/mod/core.html#allowoverride

Reply:
@jdelnano did you test my changes and read the documentation?